.gitignore

```diff
@@ -1 +1,2 @@
 target/
+Cargo.lock
```

Comment:
Per my other comment, this is not correct. https://doc.rust-lang.org/cargo/faq.html#why-do-binaries-have-cargolock-in-version-control-but-not-libraries